gcp-gmp-integration.md

```diff
@@ -0,0 +1,62 @@
+# Google Cloud Managed Service for Prometheus (GMP)
+
+## Overview
+
+Kubecost leverages the open-source Prometheus project as a time series database and post-processes the data in Prometheus to perform cost allocation calculations and provide optimization insights for your Kubernetes clusters, such as Amazon Elastic Kubernetes Service (Amazon EKS). Prometheus is a single machine statically-resourced container, so depending on your cluster size or when your cluster scales out, your cluster could exceed the scraping capabilities of a single Prometheus server. In this doc, you will learn how Kubecost integrates with [Google Cloud Managed Service for Prometheus (GMP)](https://cloud.google.com/stackdriver/docs/managed-prometheus), a managed Prometheus-compatible monitoring service, to enable the customer to monitor Kubernetes easily cost at scale.
+
+For this integration, GMP is required to be enabled for your GKE cluster with the managed collection. Next, Kubecost is installed in your GKE cluster and leverages GMP Prometheus binary to ingest metrics into GMP database seamlessly. In this setup, Kubecost deployment also automatically creates a Prometheus proxy that allows Kubecost to query the metrics from the GMP database for cost allocation calculation.
+
+{% hint style="info" %}
+This integration is currently in beta.
+{% endhint %}
+
+## Reference Resources
+
+* [Google Cloud Managed Service for Prometheus (GMP)](https://cloud.google.com/stackdriver/docs/managed-prometheus)
+* [Get started with self-deployed collection](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged)
+* [Configure a query interface](https://cloud.google.com/stackdriver/docs/managed-prometheus/query)
+
+## Instructions
+
+### Prerequisites
+
+* You have a GCP account/subscription.
+* You have permission to manage GKE clusters and GCP monitoring services.
+* You have an existing GKE cluster with GMP enabled. You can learn more [here](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#enable-mgdcoll-gke).
+
+### Installations
+
+You can use the following command to install Kubecost on your GKE cluster and integrate with GMP:
+
+```bash
+export GCP_PROJECT_ID="YOUR_GCP_PROJECT_ID"
+export CLUSTER_NAME="YOUR_GKE_CLUSTER_NAME"
+helm upgrade -i kubecost cost-analyzer/ \
+--namespace kubecost --create-namespace \
+--set prometheus.server.image.repository="gke.gcr.io/prometheus-engine/prometheus" \
+--set prometheus.server.image.tag="v2.35.0-gmp.2-gke.0" \
+--set global.gmp.enabled="true" \
+--set global.gmp.gmpProxy.projectId="${GCP_PROJECT_ID}" \
```

Comment:
Should we call out which items are in addition to the typical install process? Seems likely the 2 `global.gmp.` values are. Reader might not care though, in which case this question is unnecessary :)

Reply:
@ausmith Nice idea ! I added the description for each flag in the latest commit and fixed the typo.